LP-1760 ORA html templates for peer assessment

aliassad opened the pull request:
Added a mobile assessment application in which i have created
model for Assessment Template and made a change in Peer Assessment Mixin for writing a method for Peer Assessment Xblock.

aliassad committed:
LP-1706
Added a mobile assessment application in which i have created
model for Assessment Template and made a change in Peer Assessment Mixin for writing a method for Peer Assessment Xblock.

emraan004 commented on openassessment/mobile_assessment/models.py:
```diff
@@ -0,0 +1,13 @@
+    Model for Storing the HTML templates for ORA Peer Assessment
+    """
+    title = models.CharField(max_length=500)
+    slug = models.SlugField(max_length=500)
```
Let's remove this field

aliassad replied:
Done

qasimgulzar commented on openassessment/xblock/peer_assessment_mixin.py:
```diff
@@ -147,6 +153,17 @@ def render_peer_assessment(self, data, suffix=''):
+             Return the list of html templates.
+        """
+        html_templates_values = serializers.serialize('json', AssessmentTemplate.objects.all())
+        # return json.loads(html_templates_values)
```
remove commented code

aliassad replied:
Done, removed

emraan004 commented on openassessment/mobile_assessment/views.py:
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```
let's make sure we need this view

aliassad replied:
removed, that was for API interface which we are not using now.

aliassad committed:
LP-1760
Removed slug field and unused view file.
Fixed code formating.

ahmed-faraz46 commented on openassessment/mobile_assessment/models.py:
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class AssessmentTemplate(models.Model):
```
Let's please rename the Model, because  AccessmentTemplate sounds like its template of whole ORA assessment, but actually it's template of ORA table template.

aliassad replied:
Renamed

ahmed-faraz46 commented on openassessment/mobile_assessment/tests.py:
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Let's please not create the file if we aren't writing tests.

aliassad replied:
deleted the file

ahmed-faraz46 commented on openassessment/xblock/peer_assessment_mixin.py:
```diff
@@ -147,6 +153,15 @@ def render_peer_assessment(self, data, suffix=''):
 
         return self.render_assessment(path, context_dict)
 
+    @XBlock.handler
```
Let's please avoid the core change, one possible way is to write a mixin in our application and inherit that mixin from this one implement this particular handler in that mixin and use new mixin everywhere.

aliassad replied:
created a new mixin in our new application and removed the code from this core file

aliassad committed:
LP-1760
Added a new mixin for ORA prompt template
Code refactoring and fixed naming conventions issues.

ahmed-faraz46 commented on openassessment/xblock/peer_assessment_mixin.py:
```diff
@@ -1,5 +1,6 @@
 import logging
 
+
```
Nice work @aliassad to move `get_prompt_templates` in our app, but you've left some empty lines in this core file, would you please remove these empty lines as well?

aliassad replied:
yes removed

ahmed-faraz46 commented on openassessment/table_prompts/apps.py:
```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+
+class MobileAssessmentConfig(AppConfig):
```
As we have renamed the app let's please rename this config class as well.

aliassad replied:
Done

aliassad committed:
LP-1760
Removed empty lines and renamed the project config class

ahmed-faraz46 reviewed (approved):
LGTM 👍
